playground/custom-application-config.mjs

```diff
@@ -31,6 +31,7 @@ const config = {
   headers: {
     csp: {
       'connect-src': ['${env:HOST_GCP_STAGING}'],
+      'frame-src': ['${env:LOCAL_HOST}'],
```

Comment:
This is to allow rendering a Custom View in the Playground application.

Reply:
Shouldn't we include the `frame-src` directive in the default CSP config?

https://github.com/commercetools/merchant-center-application-kit/blob/51deaa6fc32b3be2439853d952ce8a5ae6db93e3/packages/mc-html-template/src/process-headers.ts#L72-L118

Reply:
Yes, you're right.

Since frames are expected to be loaded from the same origin as the host application, we shouldn't need to even add the policy but since we have a [very restrictive default](https://github.com/commercetools/merchant-center-application-kit/blob/main/packages/mc-html-template/src/process-headers.ts#L74), I just added this directive to the default ones to allow frames from the same origin here: c7ddc2df4bd2f6abee3ecf24e31037116e7ae695